src/types/hexColorCode.ts

```diff
@@ -0,0 +1,12 @@
+import {StrongType, makeStrong} from '../strong-type';
+
+import {STRules} from '../rules';
+
+export type StrongHexCode = StrongType<string>;
+
+export function makeHexCode(initial: string | null | undefined, fallback: string): StrongHexCode {
```

Comment:
- [ ] Update the return type to `StrongHexColorCode` to match the type changes requested here.